### PR TITLE
#1830 - Test case fix

### PR DESCRIPTION
**Pull request by dheepak-aot**
Fixing the test case failure by having the right institution in place and removing the test cases which are not valid anymore with decorators in place.

### CODE REVIEW

File: sources/packages/backend/apps/api/src/route-controllers/student/_tests_/e2e/student.institutions.controller.getStudentProfile.e2e-spec.ts [hunk 1]
```diff
@@ -88,27 +88,6 @@ describe("StudentInstitutionsController(e2e)-getStudentProfile", () => {
         sin: student.sinValidation.sin,
       });
   });
-  // TODO: When the authorization is implemented for institution student search,
```
Comment (dheepak-aot) [inline sources/packages/backend/apps/api/src/route-controllers/student/_tests_/e2e/student.institutions.controller.getStudentProfile.e2e-spec.ts:91]: This is not a valid test scenario anymore.

**Review by andrepestana-aot — APPROVED** [review]
LGTM.

**Review by ann-aot — APPROVED** [review]
Thanks for doing the fix @dheepak-aot

**Review by guru-aot — APPROVED** [review]
LGTM

**Review by andrewsignori-aot — APPROVED** [review]
👍

**Pull request merged**